ui/Screen/Org/MemberRegistration.svelte

```diff
@@ -17,19 +17,22 @@
     Transaction,
   } from "../../DesignSystem/Component";
 
+  export let params = null;
+  const orgId = params.id;
+
   let state = RegistrationFlowState.Preparation;
-  let userHandle,
-    transaction,
+  let payer,
     subject,
-    payer,
+    transaction,
+    userHandle,
     validating = false;
   const validation = memberHandleValidationStore();
 
   const next = () => {
     switch (state) {
       case RegistrationFlowState.Preparation:
         if ($validation.status === ValidationStatus.Success) {
-          transaction = registerMemberTransaction("monadic", userHandle);
+          transaction = registerMemberTransaction(orgId, userHandle);
           payer = formatPayer($session.data.identity);
```

Comment:
Member registration should be payed by the org. 
If it is outside of the scope of this PR, maybe you could add a Todo.

Reply:
See @cloudhead's comment; the avatar is intentionally not there because we haven't decided on a way to reconcile CoCo identity (which contains the user's avatar) and registry users (which does not contain any avatar information). Also the wallet work will be encompassed by the Wallets I milestone, TBD.